service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -137,7 +139,7 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
         Map<String, Object> parameters = localAuthorityEmailContentProvider
             .buildLocalAuthorityStandardDirectionOrderIssuedNotification(caseDetails, localAuthorityCode);
         String reference = Long.toString(caseDetails.getId());
-        String email = localAuthorityEmailLookupConfiguration.getLocalAuthority(localAuthorityCode).getEmail();
+        String email = inboxLookupService.getLocalAuthorityOrFallbackEmail(caseDetails,localAuthorityCode);
```

Comment:
can we make the method perform only a single task?

Reply:
I would suggest splitting it into 2 methods. The first can be called `getLocalAuthorityEmail` and could return an `Optional<String>` which contains null when no LA email is found, then chain the method with an `.orElse` and inside there call a method to get the FPLA email or whatever it is called.

Reply:
Done